server/api/rounds.py

```diff
@@ -899,29 +899,29 @@ def sample_batches(
     assert len(contest_sample_sizes) == 1
     contest, sample_size = contest_sample_sizes[0]
 
-    num_previously_sampled = (
-        SampledBatchDraw.query.join(Batch)
-        .join(Jurisdiction)
-        .filter_by(election_id=election.id)
-        .count()
-    )
-
     # Create a mapping from batch keys used in the sampling back to batch ids
     batches = (
         Batch.query.join(Jurisdiction)
         .filter_by(election_id=election.id)
-        .values(Jurisdiction.name, Batch.name, Batch.id)
+        .with_entities(Jurisdiction.name, Batch.name, Batch.id)
```

Comment:
`.values` is actually [deprecated](https://docs.sqlalchemy.org/en/14/orm/query.html#sqlalchemy.orm.Query.values) - `.with_entities` is recommended as a replacement